lib.hpp

```diff
@@ -0,0 +1,17 @@
+#ifdef _WIN32
+#define DLL_EXPORT __declspec(dllexport)
+#else
+#define DLL_EXPORT
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+DLL_EXPORT unsigned char* chunk;
```

Comment:
The internal structure to store chunks should not be exposed

Reply:
Do you mean to use it as a `private` data member, please?

Reply:
No, just that you don't need to export the `unsigned char* chunk`.
It will be written in the documentation, that the library-function `pushChunk` expects an `unsigned char*` to a memory-location on the heap. That's it.